packages/affiliates/libs/affiliates/devmining.js

```diff
@@ -77,13 +77,14 @@ module.exports = ({ queries, empAbi, coingecko, synthPrices, firstEmpDate }) =>
       console.error(empAddress, "synthetic price failed", err);
     }
     try {
+      if (fallbackPrice) {
```

Comment:
forcing fallback prices to be used before coingecko. This was requested by @smb2796 as it caused an issue last week.